storage/backup/backup-cli/src/utils/stream/try_buffered_x.rs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+///! This is a copy of `futures::try_stream::try_buffered` from `futures 0.3.16`, except that it uses
```

Comment:
It kinda sucks that we have to manage these forks (including the ones already in the codebase). Perhaps we could consider pushing this upstream, or starting a conversation with the developers to provide this functionality.

Reply:
https://github.com/rust-lang/futures-rs/issues/2528